[2025-04-08T22:31:39.323+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-04-08T22:31:40.482+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: extract_raw_data_pipeline.extract_raw_data manual__2025-04-08T22:30:16.970994+00:00 [queued]>
[2025-04-08T22:31:40.705+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: extract_raw_data_pipeline.extract_raw_data manual__2025-04-08T22:30:16.970994+00:00 [queued]>
[2025-04-08T22:31:40.706+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2025-04-08T22:31:41.311+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): extract_raw_data> on 2025-04-08 22:30:16.970994+00:00
[2025-04-08T22:31:41.447+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=217) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-04-08T22:31:41.332+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'extract_raw_data_pipeline', 'extract_raw_data', 'manual__2025-04-08T22:30:16.970994+00:00', '--job-id', '9', '--raw', '--subdir', 'DAGS_FOLDER/driven_data_pipeline.py', '--cfg-path', '/tmp/tmp1ql0jb2h']
[2025-04-08T22:31:41.448+0000] {standard_task_runner.py:72} INFO - Started process 228 to run task
[2025-04-08T22:31:41.448+0000] {standard_task_runner.py:105} INFO - Job 9: Subtask extract_raw_data
[2025-04-08T22:31:41.787+0000] {task_command.py:467} INFO - Running <TaskInstance: extract_raw_data_pipeline.extract_raw_data manual__2025-04-08T22:30:16.970994+00:00 [running]> on host 5a8aeda502d0
[2025-04-08T22:31:43.477+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='extract_raw_data_pipeline' AIRFLOW_CTX_TASK_ID='extract_raw_data' AIRFLOW_CTX_EXECUTION_DATE='2025-04-08T22:30:16.970994+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-04-08T22:30:16.970994+00:00'
[2025-04-08T22:31:43.545+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-04-08T22:31:44.494+0000] {driven_data_pipeline.py:143} INFO - Started batch processing for 2025-04-08.
[2025-04-08T22:31:44.494+0000] {driven_data_pipeline.py:34} INFO - Created synthetic data for RO country code.
[2025-04-08T22:31:47.739+0000] {job.py:229} INFO - Heartbeat recovered after 10.63 seconds
[2025-04-08T22:32:29.070+0000] {driven_data_pipeline.py:100} INFO - Written 100372 records to the CSV file.
[2025-04-08T22:32:44.794+0000] {driven_data_pipeline.py:116} INFO - Added UUID to the dataset.
[2025-04-08T22:32:44.836+0000] {driven_data_pipeline.py:151} INFO - Finished batch processing 2025-04-08.
[2025-04-08T22:32:44.837+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-04-08T22:32:44.999+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-04-08T22:32:45.045+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=extract_raw_data_pipeline, task_id=extract_raw_data, run_id=manual__2025-04-08T22:30:16.970994+00:00, execution_date=20250408T223016, start_date=20250408T223140, end_date=20250408T223244
[2025-04-08T22:32:45.935+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-04-08T22:32:47.529+0000] {taskinstance.py:3900} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-04-08T22:32:47.689+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
